build.gradle.kts

```diff
@@ -23,89 +16,74 @@ dependencies {
     // jUnit5 API
     api(platform("org.junit:junit-bom:5.3.2"))
     api("org.junit.jupiter:junit-jupiter-api")
-    testRuntime("org.junit.jupiter:junit-jupiter-engine")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 }
 
-bintray {
-    user = (project.findProperty("bintrayUser")
-            ?: System.getenv("BINTRAY_USER"))?.toString()
-    key = (project.findProperty("bintrayApiKey")
-            ?: System.getenv("BINTRAY_API_KEY"))?.toString()
-
-    setPublications("Bintray")
-
-    dryRun = (project.version as? String)?.endsWith("SNAPSHOT") ?: true
-
-    pkg.apply {
-        repo = "maven"
-        name = "compile-testing-extension"
-        setLicenses("Apache-2.0")
-        websiteUrl = "https://github.com/Kiskae/compile-testing-extension"
-        issueTrackerUrl = "https://github.com/Kiskae/compile-testing-extension/issues"
-        vcsUrl = "https://github.com/Kiskae/compile-testing-extension.git"
-
-        githubRepo = "Kiskae/compile-testing-extension"
+java {
+    withJavadocJar()
+    withSourcesJar()
 
-        version.apply {
-            name = project.version.toString()
-            released = Date().toString()
-        }
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
Can we hold off bumping to Java 17? I don't see the patch using any feature from Java 17. We can use `javac` from JDK 17 but still target JDK11. Otherwise the published jar will not be usable by projects that are currently deployed on JDK11

Reply:
Oops i'm blind, this is only setting the toolchain, not the release target. I see the release set to JDK 8 below. Thanks